go/roothash/api/commitment/pool.go

```diff
@@ -173,7 +174,7 @@ func (p *Pool) addOpenExecutorCommitment(blk *block.Block, sv SignatureVerifier,
 
 	// Verify RAK-attestation.
 	if p.Runtime.TEEHardware != node.TEEHardwareInvalid {
-		n, err := nl.Node(id)
+		n, err := nl.Node(context.TODO(), id)
```

Comment:
could encapsulate the context in the NodeLookup implementation. or if the NodeLookup needs to be long lived, could pass in context from caller

Reply:
Passing context from caller seems cleaner, added.